app/src/main/res/resource/values-de/strings.xml

```diff
@@ -56,20 +56,22 @@
     <string name="unsplash">Unsplash.com</string>
```

Comment:
I cannot comment line 46, but why "Sie". Such formal? Why not just "du" und "deine"…
We're on the internet here, so let's don't be such formal. :wink: